### ISSUE TITLE
Write unit tests for robot_control

### ISSUE THREAD

**Issue by cyrillg** [cyrillg/ros-playground#2]
Write unit tests for the _robot_control_ package.
Need to determine their granularity as well as how to implement them.

**Comment by cyrillg** [comment]
I am facing issues to trigger the _robot_control_ tests from `catkin_make run_tests`. It sometimes works fine, and other times fails with the following import error:

```
Built target tests
Removing test result files from '/home/cyrill/ros-playground/build/test_results/robot_control'
- removing '/home/cyrill/ros-playground/build/test_results/robot_control/nosetests-tests.xml'
Built target clean_test_results_robot_control
-- run_tests.py: execute commands
  /usr/bin/cmake -E make_directory /home/cyrill/ros-playground/build/test_results/robot_control
  /usr/bin/nosetests-2.7 -P --process-timeout=60 --where=/home/cyrill/ros-playground/src/robot_control/tests --with-xunit --xunit-file=/home/cyrill/ros-playground/build/test_results/robot_control/nosetests-tests.xml
E
======================================================================
ERROR: Failure: ImportError (No module named robot_control.controller_lib)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/cyrill/ros-playground/src/robot_control/tests/test_controller.py", line 7, in <module>
    from robot_control.controller_lib import *
ImportError: No module named robot_control.controller_lib

----------------------------------------------------------------------
Ran 1 test in 0.001s

FAILED (errors=1)
-- run_tests.py: verify result "/home/cyrill/ros-playground/build/test_results/robot_control/nosetests-tests.xml"
Built target _run_tests_robot_control_nosetests_tests
Built target _run_tests_robot_control_nosetests
Built target _run_tests_robot_control
Built target run_tests
```
It is still unclear what is the cause of this behaviour.
I observed that when I modify the _robot_control_ CMakeList.txt and then recompile the repo from scratch, it works.
In any case, the tests can be run by going to the root of the _robot_control_ package and running `nostests`.

**Comment by cyrillg** [comment]
Will close this for now, since this `catkin_make run_tests` issue does not seem to affect the Travis CI.

Merged in #5
